tsconfig.json

```diff
@@ -11,13 +11,17 @@
     ],
     "strict": true,
     "outDir": "./dist", /* Redirect output structure to the directory. */
-    "rootDir": "./src", 
+    "rootDirs": [
+      "src",
+      "test"
```

Comment:
Remove this and use a special test/tsconfig.json for the occasion. This will revert all path changes in api-extractor and package.json